core/domain/services/assets/CoreAssetManager.php

```diff
@@ -0,0 +1,357 @@
+<?php
+
+namespace EventEspresso\core\domain\services\assets;
+
+use EE_Currency_Config;
+use EE_Registry;
+use EE_Template_Config;
+use EEH_Qtip_Loader;
+use EventEspresso\core\domain\DomainInterface;
+use EventEspresso\core\domain\values\assets\JavascriptAsset;
+use EventEspresso\core\exceptions\InvalidDataTypeException;
+use EventEspresso\core\exceptions\InvalidEntityException;
+use EventEspresso\core\services\assets\AssetCollection;
+use EventEspresso\core\services\assets\AssetManager;
+use EventEspresso\core\services\assets\Registry;
+use EventEspresso\core\services\collections\DuplicateCollectionIdentifierException;
+use InvalidArgumentException;
+
+/**
+ * Class CoreAssetManager
+ * Manager class for for Event Espresso core assets
+ *
+ * @package EventEspresso\core\services\assets
+ * @author  Brent Christensen
+ * @since   $VID:$
+ */
+class CoreAssetManager extends AssetManager
+{
+
+    // WordPress core / Third party JS asset handles
+    const JS_HANDLE_JQUERY                = 'jquery';
+
+    const JS_HANDLE_JQUERY_VALIDATE       = 'jquery-validate';
+
+    const JS_HANDLE_JQUERY_VALIDATE_EXTRA = 'jquery-validate-extra-methods';
+
+    const JS_HANDLE_UNDERSCORE            = 'underscore';
+
+    const JS_HANDLE_ACCOUNTING_CORE       = 'ee-accounting-core';
```

Comment:
I prefixed constants with "EE_" if they were scripts that contained code written by EE. 
This particular handle is for the `accounting.js` core library, but I _think_ you had namespaced it to avoid conflicts with other plugins that may be loading a different version (or using a different set of base configurations). So I kept the EE namespacing for the actual handle but set the constant name to reflect that is is not actually Event Espresso code.